pkg/yurtmanager/controller/yurtcoordinator/podbinding/pod_binding_controller.go

```diff
@@ -149,12 +150,8 @@ func (r *ReconcilePodBinding) processNode(node *corev1.Node) error {
 
 		// pod binding takes precedence against node autonomy
 		if nodeutil.IsPodBoundenToNode(node) {
-			if err := r.configureTolerationForPod(pod, nil); err != nil {
-				klog.Errorf(Format("could not configure toleration of pod, %v", err))
-			}
-		} else {
-			tolerationSeconds := int64(defaultTolerationSeconds)
```

Comment:
if end user removes all these annotations, we need to restore the tolerationseconds for pods. so `defaultTolerationSeconds` can be used as a default tolerationseconds.